neptune/experiments.py

```diff
@@ -318,10 +377,9 @@ def send_text(self, channel_name, x, y=None, timestamp=None):
     def log_text(self, log_name, x, y=None, timestamp=None):
         """Log text data to Neptune experiment
 
-        | Alias: :meth:`~neptune.experiments.Experiment.send_text`
-        | If text log exists (determined by ``log_name``), then append new value to it.
-          If does not, create new text log and send first value to it.
-        | See :ref:`limits<limits-top>` for information about API and storage usage upper bounds.
+        If text log exists (determined by ``log_name``), then append new value to it.
+        If does not, create new text log and send first value to it.
+        See :ref:`limits<limits-top>` for information about API and storage usage upper bounds.
```

Comment:
@hubertjaworskineptune pipe was purposeful here. It makes a new line without making new paragraph.